contributing/code/core_team.rst

```diff
@@ -69,23 +65,40 @@ Active Core Members
   * **Kévin Dunglas** (`dunglas`_) can merge into the PropertyInfo_,
     Serializer_ component;
 
-  * **Abdellatif AitBoudad** (`aitboudad`_) can merge into the Translation_
-    component;
-
-  * **Jakub Zalas** (`jakzal`_) can merge into the DomCrawler_ component;
+  * **Jakub Zalas** (`jakzal`_) can merge into the DomCrawler_ and Intl_
+    components;
 
   * **Christian Flothmann** (`xabbuh`_) can merge into the Yaml_ component;
 
   * **Javier Eguiluz** (`javiereguiluz`_) can merge into the WebProfilerBundle_
-    bundle.
+    bundle;
+
+  * **Grégoire Pineau** (`lyrixx`_) can merge into the Workflow_ component;
+
+  * **Ryan Weaver** (`weaverryan`_) can merge into the Security_ component and
+    the SecurityBundle_ bundle;
```

Comment:
I mean changing "[...] the SecurityBundle_ bundle;" to ”[...] the SecurityBundle_;". This is a bit like how we deal with interface names when using `class` roles in the rest of the documentation.